lib/action_view/component/base.rb

```diff
@@ -197,6 +193,10 @@ def with_content_areas(*areas)
 
         private
 
+        def const_source_location_supported?
```

Comment:
Perhaps it would be helpful if you added documentation about this method only being in ruby 2.7+

Reply:
I added a little inline comment, good thinking!